### PR TITLE
Add interactive chart based on user's votes

### PR DESCRIPTION
**Pull request by isabelsang**
Add select form for users to vote on their favorite breakfast food
After user submits form, column chart is updated with new vote count
Still want to stylize chart, hoping to make each column diff color

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/survey-data")
+public class SurveyDataServlet extends HttpServlet {
+
+  private Map<String, Integer> bfastVotes = new HashMap<>();
```
Comment (BearBurg) [inline portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java:31]: If you want to use hashMap here as database, I think you need to make it static.

File: portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+@WebServlet("/survey-data")
```
Comment (BearBurg) [inline portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java:28]: please add javadoc for this class.

File: portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    String json = GSON.toJson(bfastVotes);
+    response.getWriter().println(json);
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String bfast = request.getParameter("bfast-survey");
+    
+    int currentVotes;
+    if(bfastVotes.containsKey(bfast)){
```
Comment (BearBurg) [inline portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java:46]: you can use `getOrDefault` instead.

File: portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java (hunk 2 above)
Comment (isabelsang) [reply portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java:28]: By javadoc do you mean a comment describing what the servlet does? Something like: 
/** Servlet that returns survey data*/
I found Javadoc in the Java style guide but just want to double check.

Comment (BearBurg) [reply portfolio/src/main/java/com/google/sps/servlets/SurveyDataServlet.java:29]: Yes. https://google.github.io/styleguide/javaguide.html#s7-javadoc

**Pull request merged**